src/blocks/gist/styles/style.scss

```diff
@@ -57,4 +57,8 @@
 		margin-top: 0.5em;
 		text-align: center;
 	}
+
+	table {
+		table-layout: auto;
+	}
```

Comment:
In terms of targeting only `.is-twentytwenty`, I am not sure that we should limit in this case. `table-layout: auto` seemed to render correctly in the different themes. What are your thoughts?

Reply:
If `table-layout: auto` doesn't cause any issues to add, then we wouldn't need to target `is-twentytwenty`. 🤘

Reply:
The TwentyTwenty theme seems to be setting `table-layout: fixed` intentionally. I don't understand why this behavior is being set but `auto` is fine (and the browser's default).